internal/pkg/service/common/configmap/visit.go

```diff
@@ -46,6 +46,8 @@ type VisitContext struct {
 	Sensitive bool
 	// Usage contains value from the "configUsage" tag, if any.
 	Usage string
+	// Usage contains value from the "configShorthand" tag, if any.
+	Shorthand string
 	// Validate contains value from the "validate" tag, if any.
```

Comment:
Added `VisitContext.Shorthand` field, it stores value from the parent, if `configmap.Value.Value` field is found.